test/integration/requests.test.ts

```diff
@@ -759,3 +759,62 @@ describe("/requests Extension", function () {
   });
 
 });
+
+
+describe("/requests Task", function () {
+  let userId1: string;
+  let userJwtToken1: string;
+
+  let taskRequest = {
+    externalIssueUrl: "https://api.github.com/repos/Real-Dev-Squad/website-my/issues/601",
+    externalIssueHtmlUrl: "https://github.com/Real-Dev-Squad/website-my/issues/601",
```

Comment:
@sahsisunny Could you please also add a test for the case where the user is authorized but the task ID is invalid or does not exist?